extensions/pom.xml

```diff
@@ -127,24 +127,6 @@
                             <failIfNoSpecifiedTests>false</failIfNoSpecifiedTests>
                         </configuration>
                     </execution>
-                    <execution>
```

Comment:
I saw @frant-hartm pointed out that [this](https://hazelcast.slack.com/archives/C01JU7ZJYGP/p1617692048135300) was the cause for the double-execution so can you guys re-check?

Reply:
Yes, coordinating with @frant-hartm, will remove even more duplicate executions.